modules/openapi-generator/src/main/resources/rust/Cargo.mustache

```diff
@@ -1,8 +1,26 @@
 [package]
 name = "{{{packageName}}}"
 version = "{{#lambdaVersion}}{{{packageVersion}}}{{/lambdaVersion}}"
-authors = ["OpenAPI Generator team and contributors"]
+authors = [{{#infoEmail}}"{{{.}}}"{{/infoEmail}}]
```

Comment:
can we add back "OpenAPI Generator team and contributors" if infoEmail is not provided in the spec?